### PR TITLE
ARTEMIS-4723 Avoid objects left on ThreadLocal from OrderedExecutorFactory

### PR DESCRIPTION
**Pull request by clebertsuconic**
co-authored: Clebert Suconic

### CODE REVIEW

File: artemis-commons/src/main/java/org/apache/activemq/artemis/utils/actors/ProcessorBase.java [hunk 1]
```diff
@@ -117,13 +112,6 @@ public int shutdownNow(Consumer<? super T> onPendingItem, int timeout, TimeUnit
       requestedShutdown = true;
       yielded = false;
 
-      if (!inHandler()) {
-         // We don't have an option where we could do an immediate timeout
-         // I just need to make one roundtrip to make sure there's no pending tasks on the loop
-         // for that I ellected one second
-         flush(timeout, unit);
```
Comment (clebertsuconic) [inline artemis-commons/src/main/java/org/apache/activemq/artemis/utils/actors/ProcessorBase.java:124]: This was for tests and it was still causing issues anyway...


so, time to go !!! more issues than benefits

**Comment by clebertsuconic** [comment]
@cardamon notice I'm keeping you as the author on the commit. as you discovered the issue.


I went further and just removed the usage. if all my tests are good I will merge this.

**Comment by cardamon** [comment]
> @cardamon notice I'm keeping you as the author on the commit. as you discovered the issue.
> 
> I went further and just removed the usage. if all my tests are good I will merge this.

Thanks!

**Comment by clebertsuconic** [comment]
even thought I wanted to get rid of the shutdown.. that represented some semantic change.. and it's impossible to measure consequences. I will have to keep the behaviour to avoid breaking use cases and causing unexpected issues.

**Pull request merged**